module/zfs/spa.c

```diff
@@ -4235,6 +4420,8 @@ spa_tryimport(nvlist_t *tryconfig)
 		    poolname) == 0);
 		VERIFY(nvlist_add_uint64(config, ZPOOL_CONFIG_POOL_STATE,
 		    state) == 0);
+		VERIFY(nvlist_add_uint64(config, ZPOOL_CONFIG_IMPORT_TXG,
+		    spa->spa_uberblock.ub_txg) == 0);
```

Comment:
It might be nice to add this to the `spa->spa_load_info` nvlist.  This is where the other `ZPOOL_CONFIG_IMPORT_*` keys get added for import and would be consistent.  User space would then need to be updated add explictly add this key/value pair to the config passed during the real import.  What do you think?